lib/init.c

```diff
@@ -362,6 +362,13 @@ void smb2_destroy_context(struct smb2_context *smb2)
         free(discard_const(smb2->workstation));
         free(smb2->enc);
 
+        if ( NULL != smb2->connect_data )
```

Comment:
Please match the style of the rest of the file.  The curly branch goes on the same line as the if-statement
, no spaces after (,  no space before ) and we don't compare != NULL. Instead this line should just be

if (smb2->connect_data) {

Reply:
done